requirements-dev.txt

```diff
@@ -0,0 +1,13 @@
+#
+# general requirements
+#
+-r requirements.txt
+
+#
+# dev requirements
+#
+
+# testing tools
+pytest==3.0.5
+pytest-cov==2.4.0
+requests_mock
```

Comment:
Requirements should have versions against them to help improve deployment repeatability.